index.html

```diff
@@ -308,6 +308,42 @@ <h2>
           </div>
         </section>
       </div>
+      <section class="contact_container">
+        <div class="contacts">
+          <div class="contacts_head">
+            <h2 class="contacts_heading">Contact me</h2>
+            <p class="contacts_text">
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </div>
+          <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
It is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) that you collect data by using the [Formspree service](https://formspree.io/html)😉. kindly set up an account and put the right link there😎.


<img width="414" alt="FORMSPRE GITHUB" src="https://user-images.githubusercontent.com/64643995/220427024-3c65cc22-6647-481c-81f3-ff1a305ea965.png">

Reply:
Hello @Ibizugbe. I hadn't paid attention to that, thanks. Kindly try it again and leave me a message 😎.